tests/eips/eip4844/test_blobhash_opcode_contexts.py

```diff
@@ -191,21 +191,6 @@ def opcode_context(yul: YulCompiler, request):
                 ),
             },
         )
-    elif test_case == "on_INITCODE":
```

Comment:
Why was this test case removed? I'm curious :D

Reply:
It was basically made impossible by https://github.com/ethereum/EIPs/pull/7100.

I think we can still add an invalidity test where verify that a block that includes a blob transaction with nil "to" is properly rejected.

Reply:
Add it here so we don't forget - https://github.com/ethereum/execution-spec-tests/issues/144